pyvista/core/utilities/geometric_objects.py

```diff
@@ -1979,9 +1979,9 @@ def Rectangle(points: MatrixLike[float] | None = None) -> PolyData:
     if np.isclose(mag_01, 0) or np.isclose(mag_02, 0) or np.isclose(mag_12, 0):
         raise ValueError('Unable to build a rectangle with less than three different points')
 
-    scalar_pdct_01_02 = np.dot(vec_01, vec_02) / min(mag_01, mag_02) ** 2  # type: ignore[call-overload]
-    scalar_pdct_01_12 = np.dot(vec_01, vec_12) / min(mag_01, mag_12) ** 2  # type: ignore[call-overload]
-    scalar_pdct_02_12 = np.dot(vec_02, vec_12) / min(mag_02, mag_12) ** 2  # type: ignore[call-overload]
+    scalar_pdct_01_02 = np.dot(vec_01, vec_02) / min(mag_01, mag_02) ** 2
```

Comment:
This is an interesting side-effect of the new `bool` <->`np.bool_` type promotions.

The previous type error reported was:
`error: No overload variant of "min" matches argument types "floating[Any]", "floating[Any]"  [call-overload]`
which suggests  that the built-in `min` function doesn't support numpy floats.

I'm not sure why this type error with floats is now no longer considered an error with the new `bool` type promotions in place.